src/main/java/uk/gov/hmcts/reform/sscs/exception/UnhandleableEventType.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sscs.exception;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+
+import static java.util.Objects.nonNull;
+
+public class UnhandleableEventType extends Exception {
```

Comment:
maybe better `UnhandleableEventTypeException ` ?

Reply:
I'll change it